packages/wix-ui-core/src/components/avatar/avatar.tsx

```diff
@@ -125,24 +136,39 @@ export class Avatar extends React.Component<AvatarProps, AvatarState> {
     delete this.img;
   };
 
+  handleKeyDown: React.KeyboardEventHandler<HTMLElement> = event => {
```

Comment:
```suggestion
  _handleKeyDown: React.KeyboardEventHandler<HTMLElement> = event => {
```

use underscore for name, otherwise autodocs will show this method as public api